src/Logic.js

```diff
@@ -1,11 +1,15 @@
 import {Vec2} from './core.js';
 import {Enemy} from './entities/all.js';
+import {CollisionRules} from './collision/CollisionRules.js';
+import {Modern} from './collision/modern.js';
 
 export class Logic {
   constructor() {
     this.running = true;
     this.level = 0;
     this.timeSinceSpawn = 0;
+    this.collision = new CollisionRules();
+    Modern.initialize(this.collision);
```

Comment:
Inject this in the constructor instead? That way we don't have to alter the Logic class to change the collision behaviour.